### PR TITLE
Add python CSV real time example

### PR DESCRIPTION
**Pull request by SteveRosam**

### CODE REVIEW

File: python/sources/demo-data/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+quixstreams==0.5.2
```
Comment (tbedford) [inline python/sources/demo-data/requirements.txt:1]: Should this be 0.5.4?

File: python/sources/demo-data/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Demo CSV Data
+
+[This project](https://github.com/quixio/quix-samples/tree/main/python/sources/demo-data) streams F1&reg; telemetry data into a topic from a recorded Codemasters&reg; F1&reg; 2019 game session.
+
+You'll have access to the speed, acceleration, breaking and other detailed data from a real F1&reg; car, rebroadcast in real time.
```
Comment (tbedford) [inline python/sources/demo-data/README.md:5]: breaking -> braking

**Review by tbedford — DISMISSED** [review]
One minor typo and one minor query, but nothing blocking, so approved.

**Comment by SteveRosam** [comment]
@tbedford can you re-approve pls. I have made the changes you mentioned.

**Pull request merged**